MathProblems/509_Fibonacci_Number.py

```
@@ -0,0 +1,50 @@
+Explanation: F(4) = F(3) + F(2) = 2 + 1 = 3.
+ 
+
+Constraints:
+
+0 <= n <= 30
+
+'''
+
+class Solution:
+    def fib(self,n:int)->int:
+        if n==1: return 1
```

Ruff (0.11.9)

Multiple statements on one line (colon)

(E701)